### ISSUE TITLE
Title underlined in Image card widget

### ISSUE THREAD

**Issue by jh1uw** [uweb/uw_wp_theme#34]
The titles of Image cards are underlined. They should be plain.

**Comment by lcaple** [comment]
This is fixed in the lastest release: https://github.com/uweb/uw_wp_theme/releases/tag/2.0.0
